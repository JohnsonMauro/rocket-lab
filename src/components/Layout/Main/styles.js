import styled from "@emotion/styled";

const MainStyles = styled.main`
  flex: 1;
`;

export { MainStyles };
