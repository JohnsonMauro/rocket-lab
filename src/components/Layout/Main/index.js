import { MainStyles } from "./styles";

const Main = ({ children }) => <MainStyles>{children}</MainStyles>;

export default Main;
