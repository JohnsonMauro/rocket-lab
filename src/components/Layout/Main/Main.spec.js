import React from 'react';
import { render } from '@testing-library/react';

import GlobalsContainer from '@test/GlobalsContainer';
import Main from './index';

describe('<Main />', () => {
  it('Renders the Main component', () => {
    const { container } = render(<Main>Sample</Main>, {
      wrapper: GlobalsContainer,
    });
    expect(container).toMatchSnapshot();
  });
});
