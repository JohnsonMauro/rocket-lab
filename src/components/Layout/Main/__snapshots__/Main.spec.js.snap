// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Renders the Main component 1`] = `
<div>
  <main
    class="css-qhyxkp-MainStyles eala1j80"
  >
    Sample
  </main>
</div>
`;
